README.md

```diff
@@ -1,6 +1,6 @@
 # RENEWLab
 
-[![Build Status](https://4489496cb62a.ngrok.io/buildStatus/icon?job=github_public_renewlab%2Fgithub-auth)](https://4489496cb62a.ngrok.io/job/github_public_renewlab/job/github-auth/)
+[![Build Status](https://8435d1ad526d.ngrok.io/buildStatus/icon?job=github_public_renewlab%2Fdevelop-uhd)](https://8435d1ad526d.ngrok.io/job/github_public_renewlab/job/develop-uhd/)
```

Comment:
I am not sure about this commit. Is it needed? May be @mindy-min-zhang can take a look?

Reply:
Must change "8435d1ad526d" to "4489496cb62a", please. If you have done a rebase with develop, you would have noticed this change. 

The change from "github-auth" to "develop-uhd" is correct. It was done automatically for Jenkins-GitHub build status badge. (FYI, the new requirement of running ./config-ci.sh did it automatically in the background.) This kind of diff will appear in all future new feature branches merging.

Reply:
Thanks @mindy-min-zhang . For future references, if I merge from the recent develop branch it seems like these lines in the README got pulled automatically so in general, how should we keep track of these build status?

Reply:
I just found a bug in my githook file that can cause a problem in this case. I will fix it today. So you can also test the merge procedure with your PR here. I will also post a procedure here after my fix.

Reply:
I have a PR #41 for @rdoosty . Once it is merged to develop, @tingjunchen please do below.

Reply:
1. Commit your changes to your current feature branch if have not done: 
$ git add <filenames>
$ git commit
2. Check out the develop branch 
$ git checkout develop
3. Fetch and merge from the remote develop to your local develop
	$ git pull origin develop
4. You will see conflicts. Resolve the conflicts by accepting the contents from the remote develop branch. 
Go back to your feature branch to reapply your commits history:  
	$ git checkout <your-feature-branch-name>
5. Then push to the remote and PS as usual.